### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by Piper-Rains**
Please check formatting, naming and logic @Pcuddles1

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (Pcuddles1) [inline prac_05/hex_colours.py:1]: make it so that I don't need to have case sensitive names. for example, skyblue should be able to work

Comment (Piper-Rains) [reply prac_05/hex_colours.py:1]: Good point. Thanks! 
Sorry for such a late code review request.

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (Pcuddles1) [inline prac_05/word_occurrences.py:1]: it doesn't count the letters in the words
  "  for letter in word:  
        length = length + 1   "
something like that could help and use length to link with the words

**Review by Pcuddles1 — COMMENTED** [review]
not bad, a bit more work on the word_occurances but overall Good Job

**Comment by Pcuddles1** [comment]
haha, DW I did mine pretty late too
________________________________

**Review by Piper-Rains — COMMENTED** [review]
All code review comments have been resolved.

**Pull request merged**